Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: astronomer
-version: 0.30.8-rc2
-appVersion: 0.30.8-rc2
+version: 0.30.8-4
```

This version is wrong.
```suggestion
version: 0.30.8-4
```